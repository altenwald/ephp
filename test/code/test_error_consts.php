<?php

print E_ALL . "\n";
print E_ALL & ~E_STRICT;
print "\n";

