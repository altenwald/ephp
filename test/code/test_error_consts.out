32767
30719
